static/js/components/SiteContentListing.tsx

```diff
@@ -1,11 +1,18 @@
-import React from "react"
+import React, { MouseEvent as ReactMouseEvent, useState } from "react"
 import { useRouteMatch, NavLink } from "react-router-dom"
+import { useRequest } from "redux-query-react"
 import { useSelector } from "react-redux"
 
-import { siteContentListingUrl } from "../lib/urls"
-import { getWebsiteDetailCursor } from "../selectors/websites"
+import SiteEditContent from "./SiteEditContent"
 
-import { ConfigItem } from "../types/websites"
+import { siteAddContentUrl, siteContentListingUrl } from "../lib/urls"
+import { websiteContentListingRequest } from "../query-configs/websites"
+import {
+  getWebsiteDetailCursor,
+  getWebsiteContentListingCursor
+} from "../selectors/websites"
+
+import { ConfigItem, WebsiteContentListItem } from "../types/websites"
 
 interface MatchParams {
   configname: string
```

Comment:
I'd suggest that we perhaps rename `configname` here too, it just doesn't seem to capture actually what this value represents to me. What do you think?

Reply:
Sure, I decided just to do a blanket rename of that so everything is the same